tst/EnergyPlus/unit/CondenserLoopTowers.unit.cc

```diff
@@ -3987,7 +3987,7 @@ TEST_F(EnergyPlusFixture, VSCoolingTowers_WaterOutletTempTest)
 
     VSTower.calculateVariableSpeedTower(*state);
     EXPECT_DOUBLE_EQ(30.0, VSTower.OutletWaterTemp);
-    EXPECT_NEAR(0.5213, VSTower.FanCyclingRatio, 0.0001);
+    EXPECT_NEAR(0.5424, VSTower.FanCyclingRatio, 0.0001);
```

Comment:
This change to an existing test was necessary because prior to the proposed changes the range used in `CoolingTower::calculateVariableTowerOutletTemp` was not capped to the maximum range for a selected model which lead to this test using a range greater than the maximum, see screenshot below.

![justification_mod_test](https://github.com/NREL/EnergyPlus/assets/5149279/8299d39a-ca5a-4e93-b86e-757819b8b9fb)

Reply:
Interesting.  And makes sense.